src/main/java/com/iridium/iridiumskyblock/database/LostItems.java

```diff
@@ -0,0 +1,51 @@
+package com.iridium.iridiumskyblock.database;
+
+import com.iridium.iridiumcore.utils.ItemStackUtils;
+import com.iridium.iridiumteams.database.DatabaseObject;
+import com.j256.ormlite.field.DataType;
+import com.j256.ormlite.field.DatabaseField;
+import com.j256.ormlite.table.DatabaseTable;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@DatabaseTable(tableName = "lost_items")
+public class LostItems extends DatabaseObject {
+    @DatabaseField(columnName = "id", generatedId = true, canBeNull = false, unique = true)
+    private int id;
+
+    @DatabaseField(columnName = "uuid", canBeNull = false)
+    private @NotNull UUID uuid;
+
+    @DatabaseField(columnName = "time")
+    private long time;
```

Comment:
Sorry, I forgot to tell you...
https://github.com/Iridium-Development/IridiumTeams/blob/master/src/main/java/com/iridium/iridiumteams/database/types/LocalDateTimeType.java
We have automatic conversions of Datetime, so you can just do

```suggestion
    private LocalDateTime time;
```